numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -3208,7 +3179,13 @@ PyArray_GetNDArrayCVersion(void)
 NPY_NO_EXPORT unsigned int
 PyArray_GetNDArrayCFeatureVersion(void)
 {
-    return (unsigned int)NPY_API_VERSION;
+    // return (unsigned int)NPY_API_VERSION;
+    /*
+     * TODO: Preliminary returning the 1.x API version, that is a lie but
+     *       allows (for the moment) downstream modules to mix and match
+     *       and us to import old matplotlib versions in our doc builds...
+     */
+    return (unsigned int)0x01000009;
```

Comment:
Not sure lying about the runtime ABI version is the best approach, but not lying comes with the baggage of breaking CI since we would need to compile downstream used in docs for now...